api/net/Areas/Subscriber/Controllers/WorkOrderController.cs

```diff
@@ -178,14 +178,14 @@ public async Task<IActionResult> RequestTranscriptionAsync(long contentId)
     [SwaggerOperation(Tags = new[] { "WorkOrder" })]
     public async Task<IActionResult> RequestAnonymousTranscriptionAsync(long contentId, int uid)
     {
+        var redirectUrl = _apiOptions.SubscriberAppUrl + "/transcribe/" + contentId;
```

Comment:
I am leaning towards updating the notification template to point to the subscriber app instead of this endpoint.  Then the page itself will make the request to this endpoint.  That would eliminate the need to have this redirect and API endpoint configuration change.

Reply:
Sounds like a good approach.
It changes quite a lot from my PR, but won't be a big deal.

Reply:
I think it makes more sense to have the email go directly to the website instead of the api.